packages/odyssey-react-mui/package.json

```diff
@@ -99,6 +99,7 @@
     "generate:icons": "yarn generate:icons:clean && yarn generate:icons:components && yarn generate:icons:index",
     "playwright-install": "NODE_TLS_REJECT_UNAUTHORIZED=0 yarn playwright install --with-deps chromium",
     "test": "NODE_ENV=test vitest run --reporter=default",
+    "generate:wcVersion": "tsx scripts/updateWebComponentVersion.ts",
```

Comment:
Can we just say `generate:webComponentVersion`?

That would help. My first go-to for "wc" is "the bathroom".